testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -1136,8 +1137,21 @@ public void getRows() throws IOException {
     createKeyspace(keyspaceName);
     createTable(keyspaceName, tableName);
 
-    String rowIdentifier = UUID.randomUUID().toString();
+    // To try to ensure we actually find the right entry, create one other entry first
     Map<String, String> row = new HashMap<>();
+    row.put("id", UUID.nameUUIDFromBytes("abc".getBytes(StandardCharsets.UTF_8)).toString());
```

Comment:
It makes UUID static (unchanging) so that ordering would remain stable for entries. Alternatively could just hard-code some UUID.

Reply:
But this test isn't testing ordering anyway. It's testing that a get by primary key returns one item.

Reply:
But I can also just return it to its former state; maybe my change is confusing.

Reply:
Changed back to random UUID since stability of key wouldn't yet guarantee specific ordering from testing perspective (and if it did, would need to add another test case to cover 2 possible orderings).

Reply:
@jeffreyscarpenter Right: what I meant by ordering was simply that ordering of entries would be stable wrt query; backend implementation shouldn't care, obviously, but in case of problems results might vary depending on physical ordering.

Or more specifically: the way I had "wrong" implementation was to just "get all", and in that case order would matter.

But talking out aloud I realized that you are right; it's not really something that really matters here. So I put back random uuid to avoid pointless change.